cluster/manifests/psp/pod_security_policy.yaml

```diff
@@ -33,4 +33,10 @@ spec:
   supplementalGroups:
     rule: RunAsAny
   volumes:
-  - '*'
+  - emptyDir
+  - awsElasticBlockStore
```

Comment:
I don't know, but I also don't think we should disable it, if someone has a reason to use it.